readme.md

```diff
@@ -100,6 +100,85 @@ alfy.output([{
 
 <img src="media/screenshot-output.png" width="694">
 
+#### matches(input, list, [item])
+
+Returns an `Array` of items in `list` that case-insensitively contains `input`.
+
+```js
+alfy.matches('Corn', ['foo', 'unicorn']);
+//=> ['unicorn']
+```
+
+##### input
+
+Type: `string`
+
+Text to match against the `list` items.
+
+##### list
+
+Type: `Array`
+
+List to be matched against.
+
+##### item
+
+Type: `string` `Function`
+
+By default it will match against the `list` items.
+
+Specify a string to match against an object property:
+
+```js
+const list = [{
+	title: 'foo'
+}, {
+	title: 'unicorn'
+}];
+
+alfy.matches('Unicorn', list, 'title');
+//=> [{name: 'unicorn'}]
+```
+
+Or [nested property](https://github.com/sindresorhus/dot-prop):
+
+```js
+const list = [{
+	name: {
+		first: 'John',
+		last: 'Doe'
+	}
+}, {
+	name: {
+		first: 'Sindre',
+		last: 'Sorhus'
+	}
+}];
+
+alfy.matches('sindre', list, 'name.first');
+//=> [{name: {first: 'Sindre', last: 'Sorhus'}}]
+```
+
+Specify a function to handle the matching yourself. The function receives the list item and input as arguments and is expected to return a boolean whether it matches:
+
+```js
+const list = [{
+	name: 'foo',
+	sub: 'bar'
+}, {
+	name: 'unicorn',
+	sub: 'rainbow'
+}];
+
+// here we do an exact match
+alfy.matches('Foobar', list, (item, input) => item.name + item.sub === input);
+//=> [{name: 'foo', sub: 'bar'}]
```

Comment:
It might not be clear that the input is lowercased by `alfy` itself. At first I thought this example was a mistake because I thought it would not match anything because `Foobar` !== `foobar`.